frontend/packages/ceph-storage-plugin/src/components/attached-devices-mode/lso-disk-inventory/state-reducer.ts

```diff
@@ -0,0 +1,83 @@
+export enum ActionType {
```

Comment:
use `typesafe-action` to create actions for each kind and just dispatch those actions.

Reply:
I am keeping it with the plugin pattern now. Lets do once for all.